src/components/propertyeditor/propertyeditor.js

```diff
@@ -16,9 +16,29 @@ angular.module('polestar')
         type: '=',
         enum: '=',
         propName: '=',
-        group: '='
+        group: '=',
+        description: '=',
+        default: '=',
+        min: '=',
+        max: '=',
+        role: '=' // for example 'color'
       },
-      link: function postLink(/*scope, element, attrs*/) {
+      link: function postLink(scope /*, element, attrs*/) {
+        scope.hasAuto = scope.default === undefined;
+        scope.automodel = { value: false };
```

Comment:
`autoModel`?    why not just `scope.auto = true/false`?  Is this something due to angular weird scoping nature?